src/languages/swift.js

```diff
@@ -126,22 +127,98 @@ export default function(hljs) {
       SINGLE_LINE_STRING("###"),
     ]
   };
+
+  // https://docs.swift.org/swift-book/ReferenceManual/LexicalStructure.html#ID412
+  const QUOTED_IDENTIFIER = {
+    begin: concat(/`/, Swift.identifier, /`/)
+  };
+  const IMPLICIT_PARAMETER = {
+    className: 'variable',
+    begin: /\$\d+/
+  };
+  const PROPERTY_WRAPPER_PROJECTION = {
+    className: 'variable',
+    begin: `\\$${Swift.identifierCharacter}+`
+  };
+  const IDENTIFIERS = [QUOTED_IDENTIFIER, IMPLICIT_PARAMETER, PROPERTY_WRAPPER_PROJECTION];
+
+  // https://docs.swift.org/swift-book/ReferenceManual/Attributes.html
+  const AVAILABLE = {
```

Comment:
AVAILABLE_ATTRIBUTE?

Reply:
Well this reads weird later:

```
  const ATTRIBUTES = [
    AVAILABLE,
    KEYWORD_ATTRIBUTE,
    USER_DEFINED_ATTRIBUTE
  ];
```

Is it an attribute or not?  Lets clear up the confusion.

Reply:
Oh I see now.  Still, lets find some way to remove the confusion. :)

Reply:
I just named it AVAILABLE_ATTRIBUTE since it's matched as part of the attributes. I guess that should work.